step-templates/f5-ltm-rest-enable-disable-member.json

```diff
@@ -0,0 +1,114 @@
+  },
+  "Parameters": [
+    {
+      "Id": "5ed914ec-ff22-436d-ac37-1ccc5216372e",
+      "Name": "WaitForConnections",
+      "Label": "Wait on Connections?",
+      "HelpText": "If checked, the deployment won't continue until current connections on the node = 0",
+      "DefaultValue": "False",
+      "DisplaySettings": {
+        "Octopus.ControlType": "Checkbox"
+      }
+    },
+    {
+      "Id": "2207ca51-ce9c-4498-8394-e0553fbcf54c",
+      "Name": "MaxWaitTime",
+      "Label": "Max. Wait Time (seconds)",
+      "HelpText": "Defaults to 5 minutes",
+      "DefaultValue": "300",
+      "DisplaySettings": {
+        "Octopus.ControlType": "SingleLineText"
+      }
+    },
+    {
+      "Id": "7f503c0b-0050-4685-a9d8-7dbc42199ac0",
+      "Name": "EnableOrDisable",
+      "Label": "LTM Status",
+      "HelpText": "State member will be put into (Enabled, Disabled, Offline)  \n\n- **Disabled** continues to process persistent and active connections. It can accept new connections only if the connections belong to an existing persistence session.  \n\n- **Offline** allows existing connections to time out, but no new connections are allowed.",
+      "DefaultValue": "Disabled",
+      "DisplaySettings": {
+        "Octopus.ControlType": "Select",
+        "Octopus.SelectOptions": "Enabled|Enable\nDisabled|Disable\nOffline|Forced Offline"
+      }
+    },
+    {
+      "Id": "e49bb08a-c72b-4bae-95ea-bb2c4c808657",
+      "Name": "PoolName",
+      "Label": "LTM Pool name",
+      "HelpText": "Name of F5 pool",
+      "DefaultValue": "pl_DummyDeployWeb",
+      "DisplaySettings": {
+        "Octopus.ControlType": "SingleLineText"
+      }
+    },
+    {
+      "Id": "50225d2a-503c-443e-85ca-8a42cba25ea3",
+      "Name": "MemberIP",
+      "Label": "LTM Member IP",
+      "HelpText": "IP Address of F5 pool member. Default will auto discover the ip address of Tentacle.",
+      "DefaultValue": "",
+      "DisplaySettings": {
+        "Octopus.ControlType": "SingleLineText"
+      }
+    },
+    {
+      "Id": "559c46da-55be-4bb7-88fd-0d17b09455c5",
+      "Name": "HostName",
+      "Label": "LTM Host name",
+      "HelpText": null,
```

Comment:
Can you provide help text for this parameter?

Reply:
updated help text in commit 46ed7b7e1ec2365b38657eca6b9fba9c293bdec9